repo/packages/A/avi/0/marathon.json.mustache

```diff
@@ -60,6 +61,13 @@
         "hostPath": "/var/run/docker.sock",
         "containerPath": "/var/run/docker.sock",
         "mode": "RO"
+      },
+      {
+        "containerPath": "unused",
+        "mode": "RW",
+        "persistent": {
+          "size": 10
```

Comment:
is this disk truly "unused"? if so then why is it in this spec? if it's actually needed (and used to store data) then I'd recommend parameterizing the size here